joint_names: [left_arm_1_joint, left_arm_2_joint, left_arm_3_joint, left_arm_4_joint, left_arm_5_joint, left_arm_6_joint, left_arm_7_joint]

# linear velocity limit [m/s] 
v_limit: 0.15  

# angular velocity limit [rad/s]
w_limit: 0.5 